data.sql

```diff
@@ -8,4 +8,25 @@ INSERT INTO animals(id, name, escape_attempts, neutered, weight_kg, date_of_birt
 VALUES (2, 'Gabumon', 2, TRUE, 8, TO_DATE('Nov 15th, 2018', 'Mon DDth, YYYY'));
 
 INSERT INTO animals(id, name,escape_attempts, neutered, weight_kg, date_of_birth)
-VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
\ No newline at end of file
+VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 5, 'Charmander', 0, FALSE, 11,TO_DATE('Feb 8th, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 6, 'Plantmon', 0, TRUE, 5.7,TO_DATE('Nov 15th, 2021', 'Mon DDth, YYYY'));
```

Comment:
- [ ] Please change the `escape_attempts` for this record to `2` instead of `0`.